test/e2e/shoot/create_update_delete.go

```diff
@@ -21,7 +21,7 @@ import (
 	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
 	"github.com/gardener/gardener/test/e2e/shoot/internal/access"
 	"github.com/gardener/gardener/test/framework"
-	shootupdatetest "github.com/gardener/gardener/test/testmachinery/shoots/update"
+	shootupdatesuite "github.com/gardener/gardener/test/suites/shoots/update"
```

Comment:
I think, `suites` is somehow misleading, as there is `testmachinery/suites` which contains full ginkgo test suites.

Maybe call it `common` or `utils`, e.g. `test/utils/shoot/update`. `test/e2e/shoot/internal/access` could then also be moved to `test/utils/shoot/access`.